app/views/submissions/_form.html.erb

```diff
@@ -63,8 +70,61 @@
 					<%= t('submission.no_manuscript_yet') %>
 				</span>
 			</div>
-	    <div class="col-12" id="userManuscript-informer">
-  	  </div>
+	    <div class="col-12 mt-2" id="userManuscript-status">
+			<div class="col-12" id="file-size-correct" class="submission-initially-hidden">
+				<i class="fas fa-check"></i>
+				<%=t('submission.correct_file_size')%>
+			</div>
+			<div class="col-12" id="file-size-too-big" class="submission-initially-hidden">
+				<i class="fas fa-exclamation-triangle"></i>
+				<%=t('submission.file_size_too_big')%>
+			</div>
+			<div class="col-12" id="file-size-way-too-big" class="submission-initially-hidden">
+				<i class="fas fa-bomb"></i>
+				<%=t('submission.file_size_way_too_big')%>
+			</div>
+			<div class="col-12" id="file-optimize" class="submission-initially-hidden">
+				<small>
+					<%= t("submission.optimization_help_html")%>
+					<%= helpdesk(t('submission.optimization_info'),true)%>
+				</small>
+				<div class="col-12 my-1">
+					<button id="userManuscript-optimize-btn"
+						class="btn btn-sm btn-primary"
+						data-tr-working="<%= t('submission.working')%>"
+						data-tr-failed="<%= t('submission.optimization_failed')%>"
+						>
+						<%= t('submission.optimize') %>
+					</button>
+				</div>
+			<!--<div class="col-1"></div>-->
```

Comment:
Is there a reason for this? If yes, maybe add a comment why, or there is the danger that it will be deleted by some one in the future who thinks it is not necessary. (I sometimes use col-11 instead of col-12 because bootstrap occasionally messes up line breaking and overflow in col-12).

Reply:
Nope I will remove it.